frontend/workflows/projectCatalog/src/details/quick-links.tsx

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import type { clutch as IClutch } from "@clutch-sh/api";
+import {
+  Card,
+  Grid,
+  Link,
+  Popper,
+  PopperItem,
+  Tooltip,
+  TooltipContainer,
+  Typography,
+} from "@clutch-sh/core";
+
+const ICON_SIZE = "32px";
+// If only a single link, then no popper is necessary
+const QuickLink = ({ link, linkGroupName, linkGroupImage }) => (
+  <Grid item key={link.name}>
+    <Tooltip title={linkGroupName}>
+      <TooltipContainer>
+        <Link href={link.url}>
+          <img width={ICON_SIZE} height={ICON_SIZE} src={linkGroupImage} alt={link.name} />
+        </Link>
+      </TooltipContainer>
+    </Tooltip>
+  </Grid>
+);
+
+// Have a popper in the case of multiple links per group
+const QuickLinkGroup = ({ linkGroupName, linkGroupImage, links }) => {
+  const anchorRef = React.useRef(null);
+  const [open, setOpen] = React.useState(false);
+
+  return (
+    <Grid item key={linkGroupName}>
+      <Tooltip title={linkGroupName}>
```

Comment:
If we don't have a name this will just show an empty dark field under the icon. We should probably not show a tooltip if there is no value here or enforce that this must be set.

Reply:
yeah we can enforce that it is set in the proto